react/components/ActionMenu/index.js

```diff
@@ -110,9 +117,11 @@ ActionMenu.propTypes = {
   align: PropTypes.oneOf(['right', 'left']),
   /** If should close the menu after clicking an option */
   shouldCloseOnClick: PropTypes.bool,
-  /** Respecting button props contract. */
-  buttonProps: PropTypes.shape({ ...Button.propTypes }),
-  /** Button icon */
+  /** Respecting ButtonWithIcon props contract. For more info, click
+   * <a href='#/Components/Forms/ButtonWithIcon'>here</a>
```

Comment:
I'm not sure if this is the proper place for `a` tags? Posting the link itself is fine